WalletWasabi/Blockchain/Blocks/SmartHeader.cs

```diff
@@ -16,27 +21,31 @@ public SmartHeader(uint256 blockHash, uint256 prevHash, uint height, DateTimeOff
 		}
 
 		Height = height;
-		BlockTime = blockTime;
+		BlockTimeSeconds = blockTimeSeconds;
```

Comment:
`EpochTime` is better because we now exactly what it means. It can be `EpochBlockTime` too. Alternatively it could be `UnixTime`.

Reply:
I like `EpochBlockTime`. Renamed.